_data/social.yml

```
@@ -299,7 +299,7 @@ websites:
       - totp
       - u2f
     doc: https://support.twitter.com/articles/20170388
-    exception: "Mobile apps do not support Hardware 2FA. Hardware 2FA requires enabling back-up method. Cannot add multiple security keys. SMS only available on select providers."
+    exception: "Mobile apps do not support Hardware 2FA. Hardware 2FA requires enabling back-up method. SMS only available on select providers."
```

Thanks for the heads up! Let's also remove this bit as it's also outdated 😉 
```suggestion
    exception: "Hardware 2FA requires enabling back-up method. SMS only available on select providers."
```

---

Ah, I wasn't paying attention that that bit, and I don't use the app anyway, so couldn't confirm the current state.